packaging/nnstreamer.spec

```diff
@@ -114,6 +128,12 @@ Requires:	nnstreamer = %{version}-%{release}
 %description tensorflow-lite
 NNStreamer's tensor_fliter subplugin of TensorFlow Lite.
 
+%package nnfw
```

Comment:
```%if 0%{?enable_nnfw} ```  is it necessary? (or enable_nnfw_r )

Question: why enable_nnfw_r (line 86) is required?

Reply:
With recent commit, ```enable_nnfw_r``` is no more required. Anyway, ```enable_nnfw``` is introduced as the nnfw-devel does not look so stable and this is Tizen-only.